release-team/release-team-selection.md

```diff
@@ -0,0 +1,99 @@
+- staffing the role themselves
+
+Ultimately, if none of these can be satisfied, responsibility falls to the future Release Team Lead and SIG Release to staff the roles.
+
+## Considerations
+
+### Timing
+
+Staff early! The Release Team should undertake the selection process at the beginning of Code Freeze, with a deadline of three days following the current release.
+
+Any staffing decisions that cannot be made within this timeframe should be escalated to SIG Release Chairs and discussed publicly.
+
+### Membership
+
+Try to ensure potential Release Team candidates:
+- can commit to the amount of time required across the release cycle
+- are enthusiastic about being on the release team
+- are members of the relevant SIGs for their role, if applicable
+- have some prior experience with contributing to Kubernetes (such as shadowing a role for a prior release)
+
+Most importantly, strive for diversity in:
+- gender identity
+- ethnicity
+- nationality
+- locality
+- company affiliation
+
+## Selection Criteria
+
+### Release Team Lead
+
+The incoming Release Team Lead _MUST_ have participated on the Release Team for 2 or more release cycles, acting in a lead (non-shadow) capacity for at least one of those cycles.
+
+Release Team Leads should be staffed, with this order of fall-through in mind:
+- the current pool of Release Team Lead shadows
+- the current pool of Release Team members
+- former Release Team members
+
+Bear in mind that these are suggestions based on precedent and a Release Team Lead may be nominated by any Release Team member, past or present.
+
+_The new Release Team Lead can be selected via lazy consensus of the current Release Team members._
+
+### Release Team Lead Shadow
+
+Same as [Release Team Lead](#release-team-lead).
+
+### Shadows
+
+Every Release Team role should seek to accommodate a set of role shadows. This creates a new avenue for code and non-code contributors alike to contribute to the project. Additionally, it seeds future release cycles with new leaders.
+
+While there isn't a strict restriction on the amount of shadows, we've found three shadows per role to be a reasonable upper bound. However, as shadowing is intended to be a mentor / mentee journey, it is important that role leads only accept an amount of shadows that they feel that can reasonably mentor within the scope of a release cycle.
+
+If there are more contributors interested in shadowing once hitting that upper bound, we welcome them to sit in on Release Team meetings in preparation for shadowing in a future release cycle.
+
+In the coming release cycles, beginning in 1.14, we will handle shadow selection using a questionnaire, as opposed to the first-come, first-served GitHub issue approach that we've adopted during previous release cycles ([1.12](https://github.com/kubernetes/sig-release/issues/167), [1.13](https://github.com/kubernetes/sig-release/issues/280), [1.14](https://github.com/kubernetes/sig-release/issues/372)).
```

Comment:
Nit: maybe drop "In the coming release cycles," and write this in the past-tense due to it's likely long shelf life.

Reply:
Nice catch. Will update.

Reply:
Fixed!